src/features/feederMachine/FeederMachineModal.tsx

```
@@ -409,7 +410,9 @@ export const FeederMachineModal: React.FC<Props> = ({
                                     icon={ITEM_DETAILS[ingredient].image}
                                     width={7}
                                   />
-                                  <span className="text-xs">{ingredient}</span>
+                                  {!hideIngredientName && (
+                                    <span className="text-xs">{ingredient}</span>
```

ESLint

[error] Replace `{ingredient}` with `⏎······································{ingredient}⏎····································`

(prettier/prettier)